test/tools/ossfuzz/abiV2ProtoFuzzer.cpp

```diff
@@ -15,15 +15,38 @@
 	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <test/EVMHost.h>
 #include <test/tools/ossfuzz/abiV2FuzzerCommon.h>
 #include <test/tools/ossfuzz/protoToAbiV2.h>
+#include <evmone/evmone.h>
 #include <src/libfuzzer/libfuzzer_macro.h>
 #include <fstream>
 
+static auto evmone = evmc::vm{evmc_create_evmone()};
```

Comment:
Can you name the type explicitly, please? Because of the different coding style, it is not clear whether `evmc::vm` is a function or a type.